src/public/js/util.js

```diff
@@ -38,6 +38,12 @@ var UTIL = (function() {
      0xb4, 0x7c, 0xb8, 0x26, 0x77, 0x99, 0xe3, 0xa5,
      0x67, 0x4a, 0xed, 0xde, 0xc5, 0x31, 0xfe, 0x18,
      0x0d, 0x63, 0x8c, 0x80, 0xc0, 0xf7, 0x70, 0x07]);
+  log3.get = function(x) {
```

Comment:
(I do care! always curious to learn -- especially about programming language implementation)

makes sense! I figured there wasn't a .push because .push would change the length of the array and it was initialized with a particular length -- suppose that's a side effect of the way it's implemented. I don't know anything about cache architecture & was wondering how big one block was so I tried to go look it up but wasn't sure... would these be caches associated with a Javascript engine, like V8 for chrome?